managedtenants/schemas/metadata.schema.yaml

```diff
@@ -69,16 +69,6 @@ properties:
       type: string
       pattern: ^[A-Za-z0-9][A-Za-z0-9-]{0,60}[A-Za-z0-9]$
     description: "Namespaces managed by the addon-operator. Need to include the TargetNamespace."
-  serviceAccount:
-    type: string
-    pattern: ^[A-Za-z0-9][A-Za-z0-9-]{0,60}[A-Za-z0-9]$
-    description: "Name of the service account created by the addon operator to authenticate to the API."
-  policyPermissions:
-    type: array
-    description: "List of permissions in an IAM policy that are required by the addon operator."
-    items:
-      type: string
-      pattern: ^[a-z0-9]{1,60}:[A-Za-z0-9]{1,60}$
```

Comment:
Did you happen to confirm that there isn't any existing addon relying on these fields `serviceAccount` and `policyPermissions`? Because if there's any addon depending on them, it would fail to operator and cause CI failures.

PS: if that's the case, then, we can keep these fields as well and mark them as deprecated, followed by creating tickets to free all the addons from relying on these fields.

Reply:
As far as we know, no addons have been using these fields (and a quick search in MT repo seems to confirm this). This was an intermediary implementation that has not been in use and never worked.

Reply:
Sounds great, then :)